### ISSUE TITLE
Which intermediates should we make? - OSM at Federation Uni

### ISSUE THREAD

**Issue by david1597** [OpenSourceMalaria/Series4#74]
Hi All, I worked with @mattodd and the Sydney team on OSM for 12 months in 2017-2018. I am back in a position to contribute to OSM again, now as a lecturer at Federation University in Australia.

It'll be the undergraduates making the contribution, initially, our third year students who are taking an organic synthesis for drug design course. In previous years they have made simple compounds during their lab component. This year we'd like them to use the same skills and techniques but make something that is ultimately useful to someone. OSM should be perfect. With time and other constraints I think final novel compounds are out of reach in this instance, but common intermediates that others could put to use would be great. 

**Are there any intermediate compounds from series 4, that are of current interest, and currently needed by anyone?** The compounds would be ready by mid-April. We can send them on wherever they are required. Any ideas @maratsydney or others currently in the lab working on this? I'll make a quick post over in the mycetoma repo too in case efforts would be better placed on that project.

**Comment by alintheopen** [comment]
Hi @david1597 - this would be great :) I'm hoping to arrange a Skype meeting in early Feb to talk about different targets for schools and undergraduates in Mycetoma, Malaria, TB and overpriced medicines and we will have a new postdoc starting in March who will be able to help with coordination too. @mattodd would early/mid Feb be a good time for you too? I will start a GitHub post to gather thoughts/contributions too. Alice

**Comment by brichcja** [comment]
Hello folks - I'm new here, but many years ago I shared undergrad supervisions with @mattodd. Nowadays I'm at the Uni of Bristol in the UK, and run our teaching labs. Much like @david1597 above, I'm wondering about getting 3rd year UGs to make stuff for the project, though this would be next academic year (from October). So, if you're talking about this, can I join in? Chris

**Comment by MFernflower** [comment]
@brichcja as far as I can tell opensourcemycetoma doesn't really need any intermediates besides one possible hypothetical one - 3‐(4‐chloro‐2‐fluorobenzoyl)pyridine

**Comment by MFernflower** [comment]
I don't think mycetoma project needs any intermediates - 3‐(4‐chloro‐2‐fluorobenzoyl)pyridine could be nice to have a few grams of for the project however

@david1597 @brichcja @fantasy121

**Comment by alintheopen** [comment]
Hi @brichcja and @david1597 and @mattodd could we possibly Skype later this week Thurs/Fri 13/14 or early the following week?

Alice

**Comment by bendndi** [comment]
> I don't think mycetoma project needs any intermediates - 3‐(4‐chloro‐2‐fluorobenzoyl)pyridine could be nice to have a few grams of for the project however
> 
> @david1597 @brichcja @fantasy121

I've just posted some thoughts on the MycetOS intermediate [over on MycetOS](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/26)

**Comment by bendndi** [comment]
> I don't think mycetoma project needs any intermediates - 3‐(4‐chloro‐2‐fluorobenzoyl)pyridine could be nice to have a few grams of for the project however
> 
> @david1597 @brichcja @fantasy121

I've just posted some thoughts on the MycetOS intermediate [over on MycetOS](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/26)

**Comment by bendndi** [comment]
@alintheopen @mattodd @david1597 @brichcja 

Appreciate this may be a bit forward of me but I'd be happy to join any call you have regarding the UG / student approaches to OSM. As you know we have similar approach at DNDi and we're currently branching out into some new projects, it would be really interesting to hear how things are going from your side and perhaps update you on where we are :-)

**Comment by david1597** [comment]
Yes I'm good for a Skype meeting. Either Thur/Fri, or Mon/Tue next week should work. @alintheopen @mattodd @brichcja @bendndi

**Comment by MFernflower** [comment]
All mycetoma related intermediate compound discussion is at
https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/26

We will need to do same with malaria or tb compounds

**Comment by brichcja** [comment]
Yes, I can skype. When do we have in mind?

**Comment by MFernflower** [comment]
@mattodd could you upload recording to opensourcemalaria youtube?

**Comment by alintheopen** [comment]
Mon/Tue/Thurs/Fri next week would be great for me if possible? Early is tricky at the moment but looks like 6-10pm Aus time are quite sociable for Switzerland and UK/
<img width="708" alt="Screen Shot 2020-02-12 at 10 27 47 pm" src="https://user-images.githubusercontent.com/2626599/74331090-067f1f80-4de7-11ea-87d9-62a7048162ce.png">

Hung may like to join and we also have two new postdocs and a new honours student in the group who might join too!

**Comment by david1597** [comment]
I could do (Aus time):
Mon 17th 9-10pm
Tues 18th Unavailable
Thur 20th 6-10pm
Fri 21st 6-10pm

**Comment by alintheopen** [comment]
Hi again,

I’m now involved in an event Mon and Thurs evening so could do 6-7 thurs or
Fri or early wed or Thursday morning.

Sorry for any inconvenience!

**Comment by brichcja** [comment]
OK, how about 7 pm Australian (8am English, 9am Swiss) on Friday 21st?

**Comment by bendndi** [comment]
I'm flying to meet with Mat in London on the Friday morning so that time is out for me, but please go ahead without me 

(Looking at the other times I'd only be available after 7pm Sydney on the Thursday; I am however able to join if you go with an early am AUS /evening Swiss time).

**Comment by alintheopen** [comment]
Will you still be with Mat on Friday evening I could do Saturday morning if
it meant we could catch you both in the same place?

**Comment by alintheopen** [comment]
Alternatively the following Sunday/Monday are good for me - would be great
to have you there!

**Comment by david1597** [comment]
Yes, would be good to have you there @bendndi. I could also do those additional times @alintheopen has listed.

**Comment by mattodd** [comment]
Sorry to be late to the thread. Would love to be at this meeting. Am meeting @bendndi midday-ish Fri 21st, but the timing is off for Aus. That Sunday night, the 23rd, is not bad, and I'm reasonably clear the Monday 24th.
One day there will be a quicker way to arrange meetings.
Later that week, around the 27th or 28th I'm going to be arranging a strategy telecon for Open Source Antibiotics. The meeting is about what to do next, rather than targets suitable for classes, but it's possible that some targets there may be well suited for crowdsourcing since they are likely to be quite simple. I still think we ought to discuss OSM and MycetOS possibilities separately beforehand, but just FYI in case we don't identify anything.

**Comment by MFernflower** [comment]
@mattodd will meetings be uploaded to youtube?

**Comment by mattodd** [comment]
@MFernflower That'd be good, yes, and easy if we use my Zoom account to run them.

**Comment by brichcja** [comment]
OK, how about Sun 23rd at 21.00 UK time / 22.00 Geneva / 08:00 Sydney (24th)?

**Comment by bendndi** [comment]
OK for me!

**Comment by alintheopen** [comment]
Hi @bendndi @brichcja @mattodd and @david1597 and anyone else of course! Are we still on for 23/24 meeting: 21.00 UK time / 22.00 Geneva / 08:00 Sydney meeting? If so, look forward to chatting in less that 12 hours. Alice

**Comment by mattodd** [comment]
Happy to if people can make it. Can start Zoom meeting if that would facilitate. Works quite well for multiple participants.

**Comment by david1597** [comment]
Yup, still good for me and happy with whatever method is suggested.

**Comment by brichcja** [comment]
Yep, still good for me.

**Comment by MFernflower** [comment]
@alintheopen @mattodd I may be able to attend as well - could I get a time in EST? I'm curious to hear how fungus project is heading

**Comment by mattodd** [comment]
OK, starting in 15. Zoom info below. I'll record unless anyone minds. I'm strictly limited to an hour, just because I have a grant deadline at midnight. I think it'd be useful if people currently involved in any crowdsourcing/schools/uni initiatives could give a quick outline of what they're up to. Slides optional.

Matthew Todd is inviting you to a scheduled Zoom meeting.

Topic: Intermediates for Crowdsourcing
Time: Feb 23, 2020 09:00 PM London

Join Zoom Meeting
https://zoom.us/j/753258555

Meeting ID: 753 258 555

One tap mobile
+19292056099,,753258555# US (New York)
+16699006833,,753258555# US (San Jose)

Dial by your location
        +1 929 205 6099 US (New York)
        +1 669 900 6833 US (San Jose)
Meeting ID: 753 258 555
Find your local number: https://zoom.us/u/avpArYQgu

**Comment by MFernflower** [comment]
Reminder from meeting: It might be possible to get re-synthesis of first paper most active compound to two steps - https://www.sigmaaldrich.com/catalog/product/aldrich/e45600

**Comment by mattodd** [comment]
Hi all,

Sorry for the delay, but the meeting can be viewed using the details below. I've raised separately (opensourceantibiotics/murligase#18) the issue of how best we should deal with these recordings, but in the meantime we need to be sure that any action items from the recording are extracted and placed below. Volunteers for doing that - essentially highlighting agreed next steps - would be _super welcome_.

Topic: Intermediates for Crowdsourcing
Start Time : Feb 23, 2020 08:58 PM

Meeting Recording:
https://ucl.zoom.us/rec/share/xPdVD5L1yjtLYafRxmPNcfJ_FLmmT6a81SUZrPpZzhs1IyhGdYLKuFSNyLTt_Wsc

**Comment by mattodd** [comment]
Some notes from the meeting. Please correct/amend/add/tag.

1) **David's students** What open projects can they research in their labs?
They have 3 full days available in lab in April. BSc level, not necessarily Chem. No NMR. Got LCMS.
(On a separate medchem course they can do more in silico things, e.g. play with Datawarrior).

2) The **choice**: Intermediates could be made. e.g. Mycetos positive control?
Or optimisation of a reaction, using LCMS.

The need for the MycetOS intermediate is not currently great because DNDi still have plenty of this compound. The synthesis of intermediates can also carry with it a responsibility to package and ship to multiple contributors, which can become expensive.

Ben talked about OSN. 3 days is optimistic for a 2-step synthesis (involved in MycetOS intermediate). But is there something simple that could be made? Crystallisation would be good to accelerate matters. What about improvement of reactions, e.g. the DMAP discovery by Dmitrij, following progress by LCMS?

3) The perennial problem of our needing an **up to date list of synthetic targets**. What to do..? Is a solution to have more of these online meetings?

4) **Alice's Breaking Good postdoc** is starting up today, who will have time for some of this coordination.

5) Alice mentions that making more OSM Series 4 structures would work.
Question is whether we need any more. We all need a **separate OSM TC** on that.
Possible OSM targets: The AI/ML competition predictions.
Also possible to do Daraprim synthesis.
Recreate over-priced medicines.
Alice's people could take on some of the task of sharing those structures in the first place.

6) Ben mentioned work with Uni Ghana, and some others (cool).
A separate Mycetoma meeting is also needed, yes.

Ben asks: what about the **virtual one-step library approach**. Enamine real space. How do we build that in? Good question. Important.

7) **Chris Adams from Bristol**. Also has students. October start. Dozen students. 3-4 months. Teams? Chris needs help in working out what to make.
Chris asks about money: can OSM take sponsorship? Answer: yes, if terms are clearly understood by both sides.

8) **Chase Smith**. Trying to find time and people. Dr Carolyn Friel is now a collaborator. Incorporating this into curriculum. So there's a constant feedstock of students coming through. In silico/informatics intro, then doing lab. Great. Also leishmaniasis targets underway.

9) Summary of OSAntibiotics mur ligase project needed, and what could be made.

(Chat window wasn't captured, sorry).
